test/integration.spec.js

```diff
@@ -68,3 +91,33 @@ test('integration : babel syntax error', t => {
     ''
   ]);
 });
+
+test('integration : webpack multi compiler : success', t => {
+
+  var logs = output.captureLogs(() => {
+    syncWebpackWithPlugin(require('./fixtures/multi-compiler-success/webpack.config'));
+  });
+
+  assert(
+    /DONE  Compiled successfully in (.\d*)ms/.test(logs),
+    'Expected logs to include \'DONE  Compiled successfully in {{number}}ms\''
+  );
+});
+
+test('integration : webpack multi compiler : module-errors', t => {
+
+  var logs = output.captureLogs(() => {
+    syncWebpackWithPlugin(require('./fixtures/multi-compiler-module-errors/webpack.config'));
+  });
+
+  assert.deepEqual(logs, [
+    ' ERROR  Failed to compile with 2 errors',
+    '',
+    'These dependencies were not found in node_modules:',
+    '',
+    '* ./non-existing',
+    '* not-found',
+    '',
+    'Did you forget to run npm install --save for them?'
+  ]);
```

Comment:
And this would have failed with a runtime error because it used to try to access `stats.compilation` on the `MultiStats` instance.